sherpa/astro/tests/test_astro_plot.py

```diff
@@ -348,3 +352,123 @@ def test_pha_model_with_gaps_977():
     # before #977 is fixed.
     #
     assert (xlo[1:] == xhi[:-1]).all()
+
+
+def test_pha_plot_with_wstat_data(caplog):
+    """Can we create a PHA data plot with WSTAT.
+
+    For PHA data, WSTAT "silently" includes the background, which
+    complicates visualization: do we subtract the background from
+    the data, add the background to the model, or something else?
+
+    At the moment we do nothing.
```

Comment:
Is that true? I thought this PR is there specifically to do something?

Reply:
This comment was added in 19cd8fa when we did in fact "do nothing". It hasn't been updated to address the change in behavior of the code.

Reply:
And fixed.